### DISCUSSION TITLE
needs-code-formatting

### DISCUSSION THREAD

**Discussion by PhilipAbed** [renovatebot/renovate#22161] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

i got a "needs-code-formatting" label on my PR
and i got so confused by it.

i was like, i'm using eslint, what kind of formatting are they asking for?

i think the name should be "format-code-comment" or something, 
it would've been more clear in my opinion

**Comment by viceice** [comment]
@rarkins @secustor this was discussed on slack and I told him to open a discussion.

i think it makes sense to rename the label.

**Reply by secustor** [reply]
Sounds good to me. 

Tough it should be clear by the label description.

**Reply by rarkins** [reply]
I'm fine with `format-code-comments`

**Reply by viceice** [reply]
ok, so we need to change the label and the action

**Reply by rarkins** [reply]
Done
